packages/flutter/lib/src/cassowary/solver.dart

```diff
@@ -153,6 +157,9 @@ class Solver {
   final List<_Symbol> _infeasibleRows = new List<_Symbol>();
   final _Row _objective = new _Row(0.0);
   _Row _artificial = new _Row(0.0);
+
+  /// A monotonically increasing value that indicates how many times the solver
+  /// has iterated.
```

Comment:
The tick count is just a debugging aid that comes in handy when dumping solver state. We can get rid of it. Will attempt that now.